Sources/PackageDescription/Target.swift

```diff
@@ -62,11 +62,12 @@ public final class Target {
     ///
     /// The URL points to a ZIP file that contains an XCFramework at its root.
     /// Binary targets are only available on Apple Platforms.
+    @available(_PackageDescription, introduced: 5.3)
     public var url: String? {
         get { _url }
         set { _url = newValue }
     }
-    public var _url: String?
+    private var _url: String?
```

Comment:
Can the underscore be removed, now that the property is private?

Reply:
Nope, it would collide with the public `url` property declared above.
There are a few other private properties of `Target` declared similarly with a leading underscore, so I thought it's a part of the code style and can be left as is. But I can certainly rename the property into something different if I'm wrong.

Reply:
> Nope, it would collide with the public `url` property declared above.

Ah, somehow I missed that.

Yes, it seems right for the underscore to stay.